### PR TITLE
IndProp: a few edits

### PR DESCRIPTION
**Pull request by yurrriq**

### CODE REVIEW

File: src/IndProp.lidr [hunk 1]
```diff
@@ -1,13 +1,13 @@
-= IndProp : Inductively Defined Propositions
+= IndProp: Inductively Defined Propositions
 
 > module IndProp
-
+>
```
Comment (clayrat) [inline src/IndProp.lidr:4]: Is this needed not to break the PDF?

File: src/IndProp.lidr [hunk 2]
```diff
@@ -54,14 +56,13 @@ We can represent a proof using these rules by combining rule applications into a
...
-                             ------ (ev_SS)
-                              ev 4
-```
+\[
+  \begin{prooftree}
+    \infer0[\idr{ev_0}]{\idr{ev 0}}
+    \infer1[\idr{ev_SS}]{\idr{ev 2}}
+    \infer1[\idr{ev_SS}]{\idr{ev 4}}
+  \end{prooftree}
+\]
```
Comment (clayrat) [inline src/IndProp.lidr:65]: 👍

File: src/IndProp.lidr [hunk 3]
```diff
@@ -162,7 +167,7 @@ Let's look at a few examples to see what this means in practice.
 
 === Pattern Matching on Evidence
 
-\todo[inline]{Edit the whole section to talk about dependent pattern matching}
+\ \todo[inline]{Edit the whole section to talk about dependent pattern matching}
```
Comment (clayrat) [inline src/IndProp.lidr:170]: Why the double slash?

**Review by clayrat — APPROVED** [review]

File: src/IndProp.lidr (hunk 3 above)
Comment (yurrriq) [reply src/IndProp.lidr:170]: The first one is a literal space, `\ `, and the second is for `\todo`. This way it doesn't try to put the note in line with the heading (which almost always falls off the page).

File: src/IndProp.lidr (hunk 1 above)
Comment (yurrriq) [reply src/IndProp.lidr:4]: No, but I like it for completely silly reasons. I've been using a cheap `awk` hack to generate `.idr` files, which help me to visualize the modules as a whole, without all the text.

```bash
$ cat >>Makefile <<EOF
%.idr: %.lidr
	@ awk '/^>/{ print substr($$0,3) }' $< >$*.idr
EOF
$ make -C src IndProp.idr
```

**Pull request merged**